src/scenes/details/Details.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import PropType from 'prop-types';
+import styles1 from '../home/Home.styles';
```

Comment:
I don't really like this approach. What do you think about calling HomeStyle, DetailsStyle or StyleHome, StyleDetails?